packages/common/src/components/Inputs/DatePickerCustom/DatePickerCustom.tsx

```diff
@@ -6,6 +6,8 @@ import { _dateReg, _format, formatDateToString, getValidDate, QuestionValidation
 import { DatePickerWrapper, StyledButton, TextInput, ValidationWrapper, Wrapper } from './Styles'
 import calendarImage from '../../../images/calendar.svg'
 import 'react-datepicker/dist/react-datepicker.css'
+import { updateClientValidationError } from '@frontend/webcert/src/store/certificate/certificateActions'
```

Comment:
We have to avoid dependencies from common to webcert. See if you can move something more to common so we can get rid this dependency to webcert.